felles/src/test/kotlin/no/nav/helsearbeidsgiver/felles/ForespurtDataTest.kt

```diff
@@ -1,11 +1,10 @@
-package no.nav.helsearbeidsgiver.inntektsmelding.helsebro.domene
+package no.nav.helsearbeidsgiver.felles
 
 import io.kotest.assertions.withClue
 import io.kotest.core.spec.style.FunSpec
 import io.kotest.data.row
 import io.kotest.matchers.collections.shouldContainExactly
```

Comment:
Vi bør ikke ha domene klasser i felles.

Reply:
Denne er ikke spesifikk til helsebro. Den er en del av responsen for trenger-endepunktet. Den ble bare lagt feil for lenge siden.